packages/types/src/base.ts

```diff
@@ -15,14 +15,13 @@ export enum ChainTypes {
   Osmosis = 'osmosis'
 }
 
-const supportedChainIds = [
-  'eip155:1',
-  'bip122:000000000019d6689c085ae165831e93',
-  'cosmos:cosmoshub-4',
-  'cosmos:osmosis-1'
-] as const
-
-export type SupportedChainIds = typeof supportedChainIds[number]
+// todo; change to object, call named chainIds
+export enum KnownChainIds {
```

Comment:
Why not change to object and call the desired name at this time?

Reply:
for static typing reasons.

Reply:
What is the comment about then? I just don't see why if we are making a breaking change and intend on it being an object with a different name we wouldn't just do that? Or delete the comment if it is irrelevant

Reply:
I removed the comment. I think it was leftover from experimenting.